src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -147,6 +148,11 @@ public int MaximumUnauthorizedExceptionsBeforeRestart
         /// </summary>
         public AzureServiceBusMessageRouterOptions Routing { get; }
 
+        /// <summary>
+        /// Gets the consumer configurable options model to change the behavior of the tracked Azure Service bus request telemetry.
+        /// </summary>
+        public MessageTelemetryOptions Telemetry => Routing.Telemetry;
+
```

Comment:
I don't see why we need this property ?
These `MessageTelemetryOptions` are already available via the `Routing` property.  Byt maybe I 'm missing some context here ?

Reply:
The `Routing.Telemetry` property is marked deprecated, it is moved to the pump options, as it is the pump solely handles telemetry. It is also an exercise in making the `MessageRouterOptions` simpler - and in the future possibly to remove it, too.

Reply:
Ok, but if `Routing.Telemetry` is marked as deprecated, doesn't the compiler emit a warning as you're calling an obsolete property now on L154 ?

Reply:
Deprecation warnings have been temporary disabled on this file, see L5: `#pragma warning disable CS0618`.

Reply:
Ok, haven't seen that.  But this means that, at some point in time the implementation of the `Telemetry` property will have to be changed ?

Reply:
Yes, when we're working on v3, we can start removing stuff and move the get/set logic a level higher.